src/main.ts

```diff
@@ -2,11 +2,19 @@ import { HexColorString } from "three";
 import { ColorizeCanvas, ColorRamp, Dataset, Track, Plotting } from "./colorizer";
 import RecordingControls from "./colorizer/RecordingControls";
 import TimeControls from "./colorizer/TimeControls";
-import UrlUtility from "./colorizer/UrlUtility";
+import {
+  CollectionEntry,
+  DEFAULT_COLLECTION_FILENAME,
+  DEFAULT_COLLECTION_PATH,
+  getCollectionData,
+  isUrl,
+  loadParamsFromUrl,
+  saveParamsToUrl,
+  getDefaultDatasetName,
+  getExpectedDatasetPath,
+} from "./colorizer/UrlUtility";
```

Comment:
Readability:  this would feel better to me if UrlUtility exported a small object that let you scope all these names at the point where you call them in this file, like `urlUtil.saveParamsToUrl` etc.  
If I'm the only one who thinks this then I will not fight for it :)

Reply:
I had it as an object export before but felt like it was too cluttered 😭 I definitely see what you mean though. @frasercl any thoughts?

Reply:
I'm generally not a huge fan of creating objects purely for code organization but I agree this is overly long. I know TypeScript has a `namespace` keyword that I've been meaning to get to know better? It could make sense to use here, if not I think I'd slightly prefer a static class.

Also there's a "util" folder in src that this file could potentially go into?

Reply:
Yeah, I was considering moving it to utils but wasn't sure if that would make the code diff messier. I'll go ahead and make both of these suggestions since I think you both have a sense of what changed.

Reply:
I don't mind namespace, exported object, or static class.  Maybe you could even do `import * as urlUtil`

Reply:
I like that last idea a fair bit actually

Reply:
Ended up just doing a wildcard import like this: `import * as urlUtils from "./utils/url_utils";`! I think this is sufficiently tidy but it can be easily refactored down the line.